doc/lua_api.txt

```diff
@@ -5141,15 +5141,28 @@ This is basically a reference to a C++ `ServerActiveObject`
   keys.
     * bit nr/meaning: 0/up, 1/down, 2/left, 3/right, 4/jump, 5/aux1, 6/sneak,
       7/LMB, 8/RMB
-* `set_physics_override(override_table)`
-    * `override_table` is a table with the following fields:
+* `set_physics_modifier(key_string, modifier_table)`: Adds a physics modifier to
+   the player under the given key. For multipliers, the product of the
+   multipliers from each physics modifier applies. For boolean values, players
+   start with the default value, but get the non-default value if any physics
+   modifier sets it to the non-default value.
+   * `key_string`: A string key identifying this modifier. Should take the format
+     "modname:modifier_name".
+   * `modifier_table`: a table with the following fields:
         * `speed`: multiplier to default walking speed value (default: `1`)
         * `jump`: multiplier to default jump value (default: `1`)
         * `gravity`: multiplier to default gravity value (default: `1`)
         * `sneak`: whether player can sneak (default: `true`)
         * `sneak_glitch`: whether player can use the new move code replications
           of the old sneak side-effects: sneak ladders and 2 node sneak jump
           (default: `false`)
+* `delete_physics_modifier(key_string)`: Removes the physics modifier set with
+  `set_physics_modifier` using key `key_string`.
+* `set_physics_override(override_table)`: Overrides the player's movement physics
```

Comment:
After using `set_physics_override()`, how do you go back to modifiers?

Looks like there's no way, maybe support `set_physics_override(nil)` to do that